docs/03-extensions/magento2/index.mdx

```diff
@@ -145,3 +145,17 @@ export default defineConfig({
   ],
 });
 ```
+
+## Known issues
+
+### Grouped product options aren't return properly on Magento 2.4.6
+
+_See related issue: https://github.com/magento/magento2/issues/37774_
+
+On Magento 2.4.6, if products that are part of a grouped product are set to "Not
+Visible Individually", they won't be returned by Magento when retrieving the
+grouped product along with its options. Instead, the returned option will be an
+array of items with `product: null`.
```

Comment:
**nitpick:** This is quite technical, but it doesn't explain the impact for the end user

**suggestion:** Add a sentence explaining the symptoms

```suggestion
On Magento 2.4.6, Front-Commerce may display a Grouped product with a price of 0€ without any options.

In this version, if products that are part of a grouped product are set to "Not
Visible Individually", they won't be returned by Magento when retrieving the
grouped product along with its options. Instead, the returned option will be an
array of items with `product: null`.
```